extensions.json

```diff
@@ -863,8 +862,8 @@
     {
       "id": "nwolverson.ide-purescript",
       "repository": "https://github.com/nwolverson/vscode-ide-purescript",
-      "version": "0.23.0",
-      "checkout": "v0.23.0",
+      "version": "0.23.1",
+      "checkout": "master",
```

Comment:
Latest release tag is still `0.23.0`. Please undo this change.

Reply:
Done!